### DISCUSSION TITLE
Create a lower resolution physics body for a heightmap

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#176] [Help / Questions]
I'm working on some procedural terrain generation and some of my terrain planes have a 256x256 resolution - is there anyway to generate a lower-resolution physics body around these planes - like say, a 16x16 physics body?

**Comment by yandeu** [comment]
I have not investigated much in terrains.
I can't think of an easy way to do it :/

**Reply by N8python** [reply]
Ok - thanks anyways. I will look into solutions.
